articles/flows/guides/regular-web-app-login-flow/add-login-using-regular-web-app-login-flow.md

```diff
@@ -17,7 +17,7 @@ This tutorial will help you add login to your regular web application using the
 
 Auth0 makes it easy for your app to implement the regular web app login flow using:
 
-* [Auth0 Mobile SDKs](/libraries): The easiest way to implement the regular web app flow, which will do most of the heavy-lifting for you. Our [Regular Web App Quickstarts](/quickstart/webapp) will walk you through the process.
+* Our [Regular Web App Quickstarts](/quickstart/webapp) will walk you through the process.
```

Comment:
This is now inconsistent with the formatting of this bullet in the other guides. Also, it doesn't form a complete sentence with the portion before the colon.

Reply:
Should we just have removed the word "Mobile"? Have we no SDKs for regular web apps?